pkg/controllers/secretstore/glob/glob.go

```diff
@@ -0,0 +1,42 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+Inspired from https://github.com/argoproj/argo-cd/tree/master/util/glob
+*/
+
+package glob
```

Comment:
Would you use the secretstore package instead of creating a new one? I don't think we need to create a new package (and file) here.